computer_vision/draw_on_webcam_video_stream.py

```diff
@@ -0,0 +1,67 @@
+import cv2
+
+
+# Variables
+drawing = False
+pt1 = (0, 0)
+pt2 = (0, 0)
+press = False
+
+
+def draw_rectangle(event, x, y, flags, params):
+
+    global pt1, pt2, drawing, press
+
+    # when mouse left button pressed
+    if event == cv2.EVENT_LBUTTONDOWN:
+        # assign top left point of rectangle
+        pt1 = (x, y)
```

Comment:
```suggestion
        top_left = (x, y)
```
Variable names that are self-documenting can make comments redundant.